doc/api/documentation.md

```diff
@@ -32,6 +32,17 @@ The stability indices are as follows:
 > [semantic versioning][] rules. Non-backward compatible changes or removal may
 > occur in any future release. Use of the feature is not recommended in
 > production environments.
+>
+> Experimental features are subdivided into stages:
+>
+> * 1.0 - Early development. Experimental features at this stage are unfinished
+>   and subject to substantial change. They are not yet ready for user testing.
+> * 1.1 - Active development. Experimental features at this stage are nearing
```

Comment:
Could we clarify here if this is ready for user testing?

Reply:
IMO everything is ready for user testing, it wouldn't make more sense to decide for our users if they want to test a WIP feature or not, it's up to them. Ready for production is a much more useful metrics IMO (and it's already documented a few line above that experimental features are not ready for production).

Reply:
> Ready for production is a much more useful metrics IMO (and it’s already documented a few line above that experimental features are not recommended for production).

I think this is the part that merits a distinction. When we have experimental features that we currently call “baking,” what we mean is that we _do_ think they’re ready for production; but we want to wait and see for a while if anyone reports issues before we officially update the docs to call them stable. But the docs say otherwise, and I don’t think this is a good thing. The features are still experimental and therefore there’s _some_ risk to using an RC experimental feature in production, but the risk is minimal enough that it should be tolerable for many people, depending on the tolerance level of the project (like a personal site might be much more willing to experiment than an eCommerce site that conducts business, for example).